src/features/user/Widget/DonationLink/DonationLinkForm.tsx

```diff
@@ -0,0 +1,244 @@
+            <div className={styles.formHeader}>
+              {t('donationLink:countryLanguageTitle')}
+            </div>
+            <InlineFormDisplayGroup>
+              <AutoCompleteCountry
+                label={t('donationLink:labelCountry')}
+                name="country"
+                defaultValue={country}
+                onChange={setCountry}
+              />
+              <MuiAutocomplete
+                id={t('donationLink:labelLanguages')}
+                options={supportedLanguages}
+                getOptionLabel={(option) =>
+                  `${option.langCode} - ${option.languageName}`
+                }
+                isOptionEqualToValue={(option, value) =>
+                  option.langCode === value.langCode
+                }
+                value={Languages}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    label="Language"
+                    placeholder={t('donationLink:Languages')}
+                  />
+                )}
+                renderOption={(props, option) => (
+                  <StyledAutoCompleteOption {...props} key={option.langCode}>
+                    <span>{`${option.langCode}`}</span>
+                    {`- ${option.languageName}`}
+                  </StyledAutoCompleteOption>
+                )}
+                onChange={(event, newLan) => setLanguage(newLan)}
+              />
+            </InlineFormDisplayGroup>
+          </div>
+          <div className={styles.singleFormGroup}>
+            <div className={styles.formHeader}>
+              {t('donationLink:projectTitle')}
+            </div>
+            <ProjectSelectAutocomplete
+              handleProjectChange={handleProjectChange}
+              project={localProject}
+              projectList={projectsList || []}
+              active={true}
+            />
+          </div>
+          <InlineFormDisplayGroup>
+            <div className={styles.formHeader}>
+              {t('donationLink:treeCounterTitle')}
+            </div>
+            <Switch
+              checked={isSupport}
+              onChange={() => {
+                setIsSupport(!isSupport);
+              }}
+              disabled={user.isPrivate}
+            />
+          </InlineFormDisplayGroup>
+          <InlineFormDisplayGroup>
+            <div className={styles.formHeader}>
+              {t('donationLink:testingTitle')}
+            </div>
+            <Switch
+              checked={isTesting}
+              onChange={() => {
+                setIsTesting(!isTesting);
+              }}
+              disabled={false}
+            />
+          </InlineFormDisplayGroup>
+          <div className={styles.singleFormGroup}>
+            <div className={styles.formHeader}>
+              {t('donationLink:urlTitle')}
+            </div>
+            <InlineFormDisplayGroup>
+              <TextField
+                id="outlined-read-only-input"
```

Comment:
1. `id` should usually be related to the purpose of the field. For example, it could be `donation-url` over here.